packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -563,6 +566,45 @@ export class TelemetryUTLogger implements ITelemetryLogger {
 }
 
 /**
+ * Take in a event object, stringify any fields that are non-primitives, and return the new event object.
+ * @param event - Event with fields you want to stringify.
+ */
+function convertToBaseEvent(event: TelemetryEventTypes): ITelemetryBaseEvent {
+    const newEvent: ITelemetryBaseEvent = {
```

Comment:
This looks weird. There is no good solution here, but I think this is better:

const newEvent = {} as ITelemetryBaseEvent;